unattended_scripts/wazuh_install.sh

```diff
@@ -330,9 +330,10 @@ function main() {
             createClusterKey
         fi
         gen_file="${base_path}/certs/password_file.yml"
-        generatePasswordFile 
-        sudo tar -zcf certs.tar -C certs/ .
-        rm -rf "${base_path}/certs"
+        generatePasswordFile
+        eval "tar -zcf '${base_path}/certs.tar' -C '${base_path}/certs/' . $debug""
```

Comment:
There are double `"` at the end. Please fix it. 
All variables starting with `$` must use the standard `${variable}`. Please change it with `debug` variable and all others inside `wazuh_install.sh`

Reply:
Done!